.github/workflows/ci.yml

```diff
@@ -251,17 +256,36 @@ jobs:
           if [[ $GITHUB_REPOSITORY =~ /(silverstripe-spellcheck|recipe-authoring-tools)$ ]] || [[ "${{ matrix.phpunit_suite }}" == "recipe-authoring-tools" ]]; then
             sudo apt install -y hunspell libhunspell-dev hunspell-en-us
           fi
-          if [[ $GITHUB_REPOSITORY =~ /(silverstripe-dynamodb)$ ]]; then
+          if [[ $GITHUB_REPOSITORY =~ /(silverstripe-dynamodb)$ ]] && [[ $MATRIX_PHPUNIT == "true" ]]; then
             # install dynamodb
+            # https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html
             mkdir _dynamodb
             cd _dynamodb
-            sudo wget https://s3.ap-southeast-1.amazonaws.com/dynamodb-local-singapore/dynamodb_local_latest.tar.gz
-            sudo tar -xf dynamodb_local_latest.tar.gz
+            wget https://d1ni2b6xgvw0s0.cloudfront.net/v2.x/dynamodb_local_latest.tar.gz
+            wget https://d1ni2b6xgvw0s0.cloudfront.net/v2.x/dynamodb_local_latest.tar.gz.sha256
+            if [[ $(shasum -a 256 dynamodb_local_latest.tar.gz) != $(cat dynamodb_local_latest.tar.gz.sha256) ]]; then
+              echo "shasum for dynamodb_local_latest.tar.gz does not match"
+              exit 1
+            fi
+            tar -xvzf dynamodb_local_latest.tar.gz
+
             # run dynamo db as background process
             # java openjdk 11 is pre-installed
             $(java -Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar) &
-            for i in {1..10}; do
-              if ! [[ $(ps -aux | grep DynamoDBLocal.jar) =~ DynamoDBLocal_lib ]]; then
+            for i in {1..20}; do
+              JAR_RUNNING=0
+              if [[ $(ps -aux | grep DynamoDBLocal.jar) =~ DynamoDBLocal_lib ]]; then
+                JAR_RUNNING=1
+              fi
+              # ping port 8000 - connection will quickly by auto closed by dynamodb
+              PORT_8000_RESPONDING=0
+              if [[ $JAR_RUNNING == 1 ]]; then
+                # telnet is preinstalled on github actions
+                if [[ $(telnet 127.0.0.1 8000) =~ 'Connected to 127.0.0.1' ]]; then
+                  PORT_8000_RESPONDING=1
+                fi
+              fi
+              if [[ $JAR_RUNNING == 0 ]] || [[ $PORT_8000_RESPONDING == 0 ]]; then
                 if [[ $i == 10 ]]; then
                   echo "dynamodb failed to start"
                   exit 1
```

Comment:
You changed this to loop up to 20 above - should this number now be 20 instead of 10?

Reply:
Updated